pulp_ansible/app/models.py

```diff
@@ -499,6 +907,19 @@ class AnsibleRepository(Repository, AutoAddObjPermsMixin):
     gpgkey = models.TextField(null=True)
     private = models.BooleanField(default=False)
 
+    sigstore_signing_service = models.ForeignKey(
+        SigstoreSigningService,
+        on_delete=models.SET_NULL,
+        related_name="ansible_repositories",
+        null=True,
+    )
+    sigstore_verifying_service = models.ForeignKey(
+        SigstoreVerifyingService,
+        on_delete=models.SET_NULL,
+        related_name="ansible_repositories",
+        null=True,
+    )
```

Comment:
I wonder if this should be a many-to-many relationship. An interesting feature of the `gpgkey` field is that the text can store multiple gpg-keys to verify against, and only one has to pass. Limiting this field to one verifying service heavily restricts the source of signatures that can be uploaded.

Reply:
Yes, I agree this would be a better approach. Is there a specific way to define many-to-many relationships in Pulp or is a `ManyToManyField` enough?

Reply:
`ManyToManyField` is enough IMO, don't think we need any extra fields in this case.